react/features/conference/route.js

```diff
@@ -113,7 +113,9 @@ function _obtainConfigHandler() {
  * @returns {void}
  */
 function _setTokenData() {
-    const localUser = APP.tokenData.caller;
+    const state = APP.store.getState();
+    const { caller } = state['features/token-data'];
+    const localUser = caller;
```

Comment:
extract the caller field with a different name instead of using an extra variable